package.json

```diff
@@ -12,9 +12,15 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^14.6.4",
+    "commander": "^6.1.0",
+    "dotenv": "^8.2.0",
     "typescript": "^4.0.2"
   },
   "bugs": {
     "url": "https://github.com/plhery/node-twitter-api/issues"
+  },
+  "dependencies": {
+    "form-data": "^3.0.0",
```

Comment:
I'd love to avoid as much as possible dependencies for this one, but for now let's keep them to iterate ☺️

Reply:
I know form-data has some nested dependencies, but handling multipart data is so much pain. 
I didn't have the motivation to implement it myself for now, but I'll be happy to do it later if it's needed 🥰